### DISCUSSION TITLE
What is the best way to change the default spacing and gap in mantine?

### DISCUSSION THREAD

**Discussion by hansyulian** [mantinedev/mantine#5863] [Q&A]
So i believe the mantine default gap is using value `md` which is 16px. Now in the design i got from designer, all the spacing and gap are like 20px. so i wonder what is the best way to change for all default without specifying `gap` in every component? currently the only one i know is using the theming default props. in case you guys have better suggestion, that would be very nice

**Comment by rtivital** [comment]
Customize `theme.spacing` – https://mantine.dev/theming/theme-object/

**Reply by hansyulian** [reply]
aside from that, it's not possible to change the default `Grid gap='md'` to `Grid gap='lg'`? because i'm not sure if the spacing only affect the grid

**Reply by hansyulian** [reply]
oh, in that case just override theme to `gap: 'lg'` right?
